### PR TITLE
appveyor: update scoop repo url, update poweshell 6 to current

### PR DESCRIPTION
**Pull request by beyondmeat**
reduce git depth.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -18,17 +18,16 @@ environment:
   innounp: '%scoop_helpers%\innounp\innounp.exe'
   matrix:
     - PowerShell: 5
-    - PowerShell: 6
+    - PowerShell: Current
```
Comment (Calinou) [inline appveyor.yml:21]: I'd prefer if we list the major version manually. This prevents the CI from suddenly breaking when upstream releases an incompatible version.

**Review by Calinou — CHANGES_REQUESTED** [review]

File: appveyor.yml [hunk 2]
```diff
@@ -39,7 +38,7 @@ for:
     - ps: . "$env:SCOOP_HOME\test\bin\test.ps1" -TestPath "$env:APPVEYOR_BUILD_FOLDER"
 - matrix:
     only:
-      - PowerShell: 6
+        - PowerShell: Current
```
Comment (Calinou) [inline appveyor.yml:41]: Same as above.

Comment (beyondmeat) [reply appveyor.yml]: I prefer to know as soon as a major version breaks and not rely on user reports, but I have made the change you requested.

**Comment by beyondmeat** [comment]
@Calinou  ready

**Pull request merged**